frontend/public/components/storage/attach-storage.tsx

```diff
@@ -65,7 +69,13 @@ export const AttachStorageForm: React.FC<AttachStorageFormProps> = (props) => {
     const newVolumeAlreadyMounted = !!volume;
     setVolumeName(newVolumeName);
     setVolumeAlreadyMounted(newVolumeAlreadyMounted);
-  }, [newPVCObj, obj, claimName, showCreatePVC]);
+
+    showCreatePVC === 'existing' && newClaimName.length > 0
```

Comment:
```suggestion
    showCreatePVC === 'existing' && newClaimName.trim().length > 0
```

Reply:
Why do we need to check the length of the name that is already created or existing? 
```suggestion
    showCreatePVC === 'existing' 
```

Reply:
@cloudbehl claim name string can be empty in the start. To avoid unnecessary api calls we are checking the length also.